app/assets/scss/overrides/_tag.scss

```diff
@@ -0,0 +1,17 @@
+.app-tag--box {
+  border: 3px solid $grey-1;
+  font-weight: bold;
+  color: $grey-1;
+  padding: 0.5em 0.75em 0.33em 0.5em;
```

Comment:
I think this is fine as a temporary override until we rejig the page, but I've never been a fan of `em` since they can be quite volatile (govuk-frontend has tried to ditch 'em entirely I think). In this case we know the parent details, so not a particular issue.

Reply:
Cool, thanks for flagging up, I'll bear that in mind in future!